lib/connection.js

```diff
@@ -230,6 +230,10 @@ var Connection = function (options, container) {
     this.closed_with_non_fatal_error = false;
 
     for (var i in aliases) {
+        if (!aliases.hasOwnProperty(i)) {
+            continue;
+        }
+      
```

Comment:
Though it doesn't show up in the github ui here, you have four spaces on this blank line which is failing the lint test.